### ISSUE TITLE
CameraFlyTo when clicked on an entity

### ISSUE THREAD

**Issue by jarnovanrhijn** [darwin-education/resium#8]
Hi, first of, great plugin so far!

So i have a question regarding the CameraFlyTo component.
I have certain entities plotted on the globe as billboards and would like to have them to fly to one whenever clicked.

i've managed to get it to work by using the raw Cesium API but would love it if this is possible with the given components. 

This is the code i would expect to work:

```
const data = [
  { lat: 48.8567, lng: 2.3508 },
  { lat: 51.507222, lng: -0.1275 }
];

export default class Cesium extends React.PureComponent<void, void> {
  flyTo(entity) {
      const destination = entity.position.getValue();
      <CameraFlyTo duration={2} destination={{...destination}} onComplete={(e) => {
        console.log('completed flyTo');
      }} />
  }

  renderEntities(): any {
      return data.map(marker => (
        <Entity
            name="name"
            description="desc"
            position={Cartesian3.fromDegrees(marker.lng, marker.lat)}
            billboard={{ image: Marker,  width: 30, height: 46 }}
          />
      ));
  }

  render() {
    return (
      <Viewer
        full
        baseLayerPicker={false}
        infoBox={false}
        homeButton={false}
        animation={false}
        timeline={false}
        scene3DOnly={true}
        shadows={true}
        fullscreenButton={false}
        onSelectedEntityChanged={e => this.flyTo(e)}
        selectionIndicator={false}
      >
        <ScreenSpaceCameraController
          maximumZoomDistance={2000}
          minimumZoomDistance={50000000}
          enableZoom={false}
        />
        <Camera maximumZoomFactor={2000} defaultZoomAmount={50000000} />
        <CustomDataSource name="custom">
          {this.renderEntities()}
        </CustomDataSource>
      </Viewer>
    );
  }
}
```

**Comment by bevikram** [comment]
You can try to set a simple CameraFlyTo component and change destination based on click and updating the state

**Comment by rot1024** [comment]
Hi, I've released v1.0.0. You can use `onClick` prop on Entity component. 

For details, refer to [documentaiton](https://resium.darwin-education.com/components/Entity). 

```jsx
class Component extends React.PureComponent {
  viewer = undefined;

  render() {
    return (
      <Viewer full ref={e => { viewer = e ? e.cresiumElement : undefined; }}>
        <Entity onClick={(moment, entity) => { viewer.zoomTo(entity); }} />
      </Viewer>
     );
  }
}
```
